.azure-pipelines/official-build-vs-with-test.yml

```diff
@@ -0,0 +1,148 @@
+# This pipeline generates daily, successful virtual SONiC images, primarily for PR testing in non-build repositories.
+
+name: $(Build.DefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)
+
+schedules:
+  - cron: "0 */8 * * *" # At 0 minutes past the hour, every 8 hours, every day UTC
+    displayName: "Daily Build and Test SONiC virtual images"
+    branches:
+      include:
+        - master
+        - 202???
```

Comment:
Build for all "202???" branches?

Reply:
yeah, or hard code all release branches?

Reply:
fixed with explicitly specifying branches.